docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -1,4 +1,10 @@
-# Reuse pipelines with namespaces
+# Reuse pipelines and group nodes with namespaces
+
+In this section, we introduce namespaces - a powerful tool for grouping and isolating nodes. Namespaces are useful in two key scenarios:
+
+- **Reusing a Kedro pipeline:** If you need to reuse a pipeline with some modifications of inputs, outputs or parameters, Kedro does not allow direct duplication because all nodes within a project must have unique names. Using namespaces helps resolve this issue by isolating identical pipelines while also enhancing visualization in Kedro-Viz.
```

Comment:
We use UK spelling instead of US in the docs (actually Vale left a comment on that too, so it's worth not just ignoring that check) 

```suggestion
- **Reusing a Kedro pipeline:** If you need to reuse a pipeline with some modifications of inputs, outputs or parameters, Kedro does not allow direct duplication because all nodes within a project must have unique names. Using namespaces helps resolve this issue by isolating identical pipelines while also enhancing visualisation in Kedro-Viz.
```